### PR TITLE
fix: resolve manual OTP fetch issue

### PR DESCRIPTION
**Pull request by Dami-18**
OTP can now be manually fetched from email. Still automatic otp fetching support left to be added

### CODE REVIEW

File: gyfe.py [hunk 1]
```diff
@@ -9,7 +9,24 @@
 import re
 import json
 
-DEPT = erpcreds.ROLL_NUMBER[2:4]
+roll = input("Roll number: ")
+DEPT = roll[2:4]
+passw = input("Password: ")
```
Comment (ashwinpra) [inline gyfe.py:14]: The roll no., password, and security question answers would already be stored in `erpcreds.py`, so no need to ask the user to enter it again

File: gyfe.py [hunk 2]
```diff
@@ -9,7 +9,24 @@
...
+    }
+session = requests.Session()
+sessionToken = erp.get_sessiontoken(session) 
+secret_question = erp.get_secret_question(headers, session, roll)
+secret_answer = input(secret_question+": ")
+loginDetails = erp.get_login_details(roll, passw, secret_answer, sessionToken)
+erp.request_otp(headers=headers, session=session, login_details=loginDetails, log=False) 
+otp = input("Enter OTP sent on mail: ")
+loginDetails["email_otp"] = otp
+ssoToken = erp.signin(headers, session, loginDetails)
```
Comment (ashwinpra) [inline gyfe.py:29]: This works, but all this is already done in the backend when you use `erp.login()`, the issue was actually with the `iitkgp-erp-login` package like Bikram told, so you just need to update the library and it should be fixed I guess :)

File: gyfe.py (hunk 1 above)
Comment (Dami-18) [reply gyfe.py:14]: Ok. I think this will be useful for web interface

File: gyfe.py (hunk 2 above)
Comment (Dami-18) [reply gyfe.py:29]: How do I update the library?

File: gyfe.py (hunk 1 above)
Comment (ashwinpra) [reply gyfe.py]: Not exactly, but yes you will have to change the method for the web interface. For example, you could convert the whole fetching code into a function that takes the roll number and password as input, and the data that the user enters in the frontend would be passed to this function in the backend.

**Review by ashwinpra — APPROVED** [review]
LGTM

File: gyfe.py (hunk 1 above)
Comment (Dami-18) [reply gyfe.py]: Ok got it

**Pull request merged**